src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,11 @@
+body,
+div {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  height: 100%;
```

Comment:
dont think you need this
```suggestion

```

Reply:
It was needed to align the calendar block vertically

Reply:
my bad then, apologies